src/BaGet.Core.Server/Configuration/ConfigureForwardedHeadersOptions.cs

```diff
@@ -1,15 +1,31 @@
+using BaGet.Core;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.HttpOverrides;
 using Microsoft.Extensions.Options;
+using System;
```

Comment:
Please place put `using System*` first. Visual Studio can do this automatically if you enable the option `Place 'System' directives when sorting usings` under `Text Editor` > `C#` > `Advanced`.

Reply:
Okay i will change that in next commit. I wasn't aware of that option.